src/main/java/core/basesyntax/strategy/impl/FruitStrategyImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.FruitService;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStrategyImpl {

```

Comment:
```suggestion
public class FruitStrategy {
```
this is not an implementation) move it to the strategy folder